test_scripts/API/VehicleData/commonVehicleData.lua

```diff
@@ -321,23 +351,72 @@ function m.processRPCSubscriptionSuccess(pRpcName, pData)
   else
     respData = pData
   end
+  local mobResParams = {
+    [respData] =  {
+      resultCode = "DISALLOWED",
+      dataType = m.allVehicleData[pData].type
+    },  
+    ["success"] = false,
+    ["resultCode"] = "DISALLOWED",
+    ["info"] = "'stabilityControlsStatus' disallowed by policies."
+  }
+  local cid = m.getMobileSession():SendRPC(pRpcName, reqParams)
+  m.getHMIConnection():ExpectRequest("VehicleInfo." .. pRpcName, reqParams):Times(0)
+  m.getMobileSession():ExpectResponse(cid, mobResParams)
+  m.getMobileSession():ExpectNotification("OnHashChange") :Times(0)
+end
+
+function m.processRPCSubscriptionPartiallyDisallowed(pRpcName, pAllowedData, pDisallowedData)
+  local mobReqParams = {
+    [pAllowedData] = true,
+    [pDisallowedData] = true
+  }
+  local hmiReqParams = {
+    [pAllowedData] = true
+  }
+  local respData
+  if pAllowedData == "clusterModeStatus" then
+    respData = "clusterModes"
+  else
+    respData = pAllowedData
+  end
   local hmiResParams = {
     [respData] = {
       resultCode = "SUCCESS",
-      dataType = m.allVehicleData[pData].type
+      dataType = m.allVehicleData[pAllowedData].type
     }
   }
-  local cid = m.getMobileSession():SendRPC(pRpcName, reqParams)
-  m.getHMIConnection():ExpectRequest("VehicleInfo." .. pRpcName, reqParams)
+  local cid = m.getMobileSession():SendRPC(pRpcName, mobReqParams)
+  m.getHMIConnection():ExpectRequest("VehicleInfo." .. pRpcName, hmiReqParams)
   :Do(function(_, data)
       m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", hmiResParams)
     end)
   local mobResParams = m.cloneTable(hmiResParams)
-  mobResParams.success = true
-  mobResParams.resultCode = "SUCCESS"
+  mobResParams = {
+    [pDisallowedData] = {
+      resultCode = "DISALLOWED",
+      dataType = m.allVehicleData[pDisallowedData].type
+    },
+    ["success"] = true,
+    ["resultCode"] = "SUCCESS",
+    ["info"] = "'stabilityControlsStatus' disallowed by policies."
+  }
   m.getMobileSession():ExpectResponse(cid, mobResParams)
 end
 
+function m.processRPCSubscriptionGenericError(pRpcName, pReqParams, pHmiResParams)
+  local reqParams = {}
+  for k,v in pairs(pReqParams) do
```

Comment:
@LitvinenkoIra  unused k
```suggestion
  for _,v in pairs(pReqParams) do
```

Reply:
@ydementieiev Done in 69c60b2146aef154480b0ad4d6d4dcfa593a54df

Reply:
@ydementieiev Done in 69c60b2